ds_judgements_public_ui/templates/pages/transactional_licence.html

```diff
@@ -5,207 +5,141 @@
   </p>
+  <h2>Before you start</h2>
+  <h3>1. Check you need to apply</h3>
   <p>
-    The Crown holds rights (copyright and database rights) in Court Judgments and Tribunal Decisions. In order to
-    re-use the information you need permission from the Keeper of Public Records, as the Crown’s officeholder
-    under Letters Patent, for the re-use of this material. <a href="{% url "open_justice_licence" %}">Open Justice Licence</a>
-    provides standard terms for re-use. If the purpose of the re-use cannot be met within the terms of the Open
-    Justice Licence, re-users need to apply
-    for a specific licence from The National Archives.
+    You only need to apply for a licence if you intend to use computational analysis in your re-use of Find
+    Case Law records in ways that are not already permitted under the Open Justice Licence.
+    The Open Justice licence permits certain re-uses without the need to apply for an additional licence, for example, when you want to:
   </p>
-  <h3>Exclusions from the licence</h3>
+  <ul>
+    <li>copy, publish, distribute and transmit the Information;</li>
+    <li>
+      exploit the Information commercially, for example, by combining it with other Information,
+      or by including it in your own product or application.
+    </li>
+  </ul>
   <p>
-    We will licence the information ‘as is’ and exclude all representations, warranties, obligations and liabilities
-    in relation to the information to the maximum extent permitted by law.
+    More information can be found:  <a href="https://caselaw.nationalarchives.gov.uk/open-justice-licence">Open Justice Licence - Find case law (nationalarchives.gov.uk)</a>.
   </p>
+  <h3>2. Check whether your application is likely to be considered high risk</h3>
+  <p>Computational analysis of judgments is currently considered high-risk when it is used to:</p>
+  <ul>
+    <li>produce fully automated legal advice; or</li>
+    <li>
+      perform automation to anticipate legal decisions directly for a client or consumer, such as
+      tools which set expectations for case outcomes through modelling or predictive analytics; or
+    </li>
+    <li>
+      otherwise directly inform or influence the decision of a third-party whether to pursue justice
+      or legal action.
+    </li>
+  </ul>
+  <h3>3. Make sure your re-use complies with all existing regulations</h3>
   <p>
-    We will not be liable for any errors or omissions in the information. We will not be liable for any loss, injury
-    or damage of any kind caused by its use.
+    Examples include the Legal Services Act (LSA), and UK General Data Protection Regulation (UKGDPR).
+    Please note that this licence:
   </p>
-  <h3>Personal data</h3>
+  <ul>
+    <li>is not a data sharing agreement for personal data is not a processing agreement for personal data</li>
```

Comment:
Should these be two separate bullet points?

Reply:
@jacksonj04 I will speak to Rose as it is based on her copy - then update accordingly -Thank you for spotting !